CompileSharpmake.bat

```diff
@@ -93,4 +93,4 @@ goto end
 
 @REM -----------------------------------------------------------------------
 :end
-exit /b %ERROR_CODE%
+exit /b %ERROR_CODE%
```

Comment:
Looks like an empty line was removed at end of file. can you restore this useless change?